OsuPlayer/Windows/FluentAppWindowViewModel.cs

```diff
@@ -78,6 +79,9 @@ public float BackgroundBlurRadius
         set => this.RaiseAndSetIfChanged(ref _backgroundImage, value);
     }
 
+    public bool IsNonLinuxOs => !RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
+    public bool IsLinuxOs => RuntimeInformation.IsOSPlatform(OSPlatform.Linux);
```

Comment:
Can you make these two a one-time evaluated property instead?

Reply:
But then we need to negate them with a converter for Example if we only have `IsLinuxOs` we need to use a converter for the Visibility when it is a non Linux OS

Reply:
No this is not what I meant

Reply:
Ohh I see now instead of Expression (=>)

Assigning those values in the `ctor`

Reply:
You can just use = {get;} = xxx;

Reply:
I know, but not a fan of using Methods and Expressions in Non Expression Bodies. `ctor` seems like a better place for that